### PR TITLE
feat: set up monorepo

### PR DESCRIPTION
**Pull request by shamsartem**

### CODE REVIEW

File: .github/workflows/pack.yml [hunk 1]
```diff
@@ -143,25 +145,30 @@ jobs:
...
 
       - name: Promote fluence-cli windows
         if: inputs.channel != 'null' && inputs.platform == 'win32-x64'
+        working-directory: apps/cli
         run: yarn oclif promote -t ${{ inputs.platform }} --version "$(jq .[] -r .github/release-please/manifest.json)" --sha "$(git rev-parse --short HEAD)" --channel ${{ inputs.channel }} --no-xz --win --indexes
 
       - name: Rename archive
         if: inputs.platform != 'win32-x64'
-        run: mv dist/fluence-*${{ inputs.platform }}*.tar.gz fluence-cli-${{ inputs.platform }}.tar.gz
+        run: mv apps/cli/dist/fluence-*${{ inputs.platform }}*.tar.gz ./fluence-cli-${{ inputs.platform }}.tar.gz
```
Comment (shamsartem) [inline .github/workflows/pack.yml:171]:
```suggestion
        run: mv apps/cli/dist/fluence-*${{ inputs.platform }}*.tar.gz fluence-cli-${{ inputs.platform }}.tar.gz
```

File: .github/workflows/pack.yml [hunk 2]
```diff
@@ -179,7 +186,7 @@ jobs:
 
       - name: Rename windows installer
         if: inputs.platform == 'win32-x64'
-        run: mv dist/win32/fluence-*.exe fluence-cli-${{ inputs.platform }}.exe
+        run: mv apps/cli/dist/win32/fluence-*.exe ./fluence-cli-${{ inputs.platform }}.exe
```
Comment (shamsartem) [inline .github/workflows/pack.yml:189]:
```suggestion
        run: mv apps/cli/dist/win32/fluence-*.exe fluence-cli-${{ inputs.platform }}.exe
```

File: .github/workflows/snapshot.yml [hunk 3]
```diff
@@ -154,14 +158,16 @@ jobs:
...
 
       - name: Pack fluence-cli
         run: yarn pack-ci
+        working-directory: apps/cli
 
       - name: Rename archives
         run: |
           for p in linux-x64 darwin-arm64; do
-            mv dist/fluence-*${p}*.tar.gz fluence-cli-${p}.tar.gz
+            mv apps/cli/dist/fluence-*${p}*.tar.gz ./fluence-cli-${p}.tar.gz
```
Comment (shamsartem) [inline .github/workflows/snapshot.yml:170]:
```suggestion
            mv apps/cli/dist/fluence-*${p}*.tar.gz fluence-cli-${p}.tar.gz
```

File: .github/release-please/config-backport.json [hunk 4]
```diff
@@ -3,7 +3,7 @@
   "release-type": "node",
   "versioning": "always-bump-patch",
   "packages": {
-    ".": {
+    "apps/cli": {
```
Comment (nahsi) [inline .github/release-please/config-backport.json:6]: This way only changes in apps/cli/* will trigger release. Is that desirable?

Comment (shamsartem) [reply .github/release-please/config-backport.json:6]: for now yeah, we don't want to release anything else. all other packages will be internal

**Review by nahsi — APPROVED** [review]

**Pull request merged**